neqo-http3/src/connection.rs

```diff
@@ -604,12 +611,14 @@ impl Http3Connection {
         stream_id: u64,
         send_stream: SendMessage,
         recv_stream: Box<dyn RecvStream>,
+        priority: Priority,
     ) {
         if send_stream.has_data_to_send() {
             self.streams_have_data_to_send.insert(stream_id);
         }
         self.send_streams.insert(stream_id, send_stream);
         self.recv_streams.insert(stream_id, recv_stream);
+        self.stream_priorities.insert(stream_id, priority);
```

Comment:
Priority *signaling* is a property of a recv stream, isn't it?  So this should apply to push streams also.

Reply:
Note: I suggested that push stream are less important at the moment, because they are not used in Firefox at the moment.
You are right that Property should be a property of a RecvStream.